code/backend/pages/02_Explore_Data.py

```diff
@@ -1,13 +1,16 @@
 import streamlit as st
 import os
-import json
 import traceback
-import pandas as pd
 import sys
-from batch.utilities.helpers.AzureSearchHelper import AzureSearchHelper
 from dotenv import load_dotenv
+from batch.utilities.helpers.EnvHelper import EnvHelper
+from backend.pages.utilities.IntegratedVectorizationSearchHandler import (
+    IntegratedVectorizationSearchHandler,
+)
+from backend.pages.utilities.AzureSearchHandler import AzureSearchHandler
 
 sys.path.append(os.path.join(os.path.dirname(__file__), ".."))
+env_helper: EnvHelper = EnvHelper()
 
 load_dotenv()
```

Comment:
I know not your actual change - but I wonder if this `load-dotenv()` is even used anymore? I would have thought it should have been before we load the envhelper

Reply:
`load_dotenv()` actually happens within the `EnvHelper` too. So probably not required here at all.

Reply:
Removed load_dotenv